crypto/s2n_openssl.h

```diff
@@ -25,7 +25,11 @@
  */
 #if defined(LIBRESSL_VERSION_NUMBER) && (OPENSSL_VERSION_NUMBER == 0x20000000L)
```

Comment:
Is `OPENSSL_VERSION_NUMBER` still set to `0x20000000L` in recent versions?

Reply:
Yes

Reply:
![image](https://user-images.githubusercontent.com/30806/199673704-e5b998ed-ec09-4286-8d54-1ab89ed43099.png)
This is from a 3.6.1 build, note the comment about "These will never change".